markdown/developers/modules.md

```diff
@@ -306,9 +306,9 @@ Once you<span class="x x-first x-last"> are </span>familiar with the module subm
 
 The key words "MUST", "MUST NOT", "SHOULD", etc. are to be interpreted as described in [RFC 2119](https://tools.ietf.org/html/rfc2119).
 
-### General
+### 1. General
```

Comment:
Personally I don't love having numbers in headings, for one it breaks all existing softlink URLs if we need to add a new heading. Also I think it's just a bit ugly 😅 

But if I'm the only one thinking this then I'm happy to go with the crowd 👍🏻 Only a very minor thing.